File: CPP/Day13.cpp
/*

Day 13: Program to find Fibonacci series of n numbers using recursion.

*/

//Program
#include <bits/stdc++.h>
using namespace std;
int fib(int n) {
    if (n <= 1) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

int main() {
    int n;
    cout<<"Enter the number: ";
    cin >> n;

    cout<<"Fibonacci series of " << n <<" is: \n";
    for (int i = 0; i < n; i++) {
        cout << fib(i) << " ";
    }
    cout << endl;
    return 0;
}



/*


Input & Output: 

Enter the number: 20
Fibonacci series of 20 is:
0 1 1 2 3 5 8 13 21 34 55 89 144 233 377 610 987 1597 2584 4181

*/

File: Python/Day13.py
""" 

Day 13:	Program to find Fibonacci series of n numbers using recursion.

"""

#Program
def fibo(n):
   if n <= 1:
       return n
   else:
       return(fibo(n-1) + fibo(n-2))

n = int(input("Enter the number: "))

if n <= 0:
   print("Plese enter a positive integer")
else:
   print("Fibonacci sequence:")
   for i in range(n):
       print(fibo(i), end = " ")
       
       
       

"""

Input and Output:

Enter the number: 5
Fibonacci sequence:
0 1 1 2 3

"""